spec/requests/cinema_halls_request_rspec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'CinemaHalls', type: :request do
+  describe 'GET /cinema_halls' do
+    it 'returns http success' do
+      get '/cinema_halls'
+      expect(response).to have_http_status(200)
```

Comment:
The test would be better if you created a `let` with at least 1 `CinemaHall`. Then you could create several `it` to check if the request is returning a proper response status and proper number of records. Remember to create one `expect` per `it`.

The current test is just checking the status of the request. Your database could have many `CinemaHall` and still return an empty array that would have status code `200` while you should expect non empty array :)

Reply:
Thanks for your comment :) I made changes in my code 👩‍💻